Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -147,6 +182,14 @@ grep -Fxq "$exportline" ~/.profile 2>/dev/null || {
     fi
 }
 
+if [[ ! -v SKIP_AP_STM_ENV ]]; then
+  if maybe_prompt_user "Register STM32 Toolchain for ccache [N/y]?" ; then
```

Comment:
Perhaps we could move the promptings up to the top and set `SKIP_AP_STM_ENV` etc instead.

Changing them from `SKIP_` to (e.g.) `DO_` might also be a nice change.

Reply:
if we move the registering, it is better to remove the prompting otherwise we will lose the ability to skip the downloading of the STM toolchain

Reply:
Ok I have move this.
Is it better for you ? If the variable doesn't exist it ask user with the maybe_prompt command otherwise it check the variable

Reply:
I have also change to DO for this variable. 
I didn't got for the DO command to have them optional and keep installing everything by default.
We could convert everyone to DO_ and use the maybe_prompt for all to setup the default value. What do you think ?